system/Validation/Validation.php

```diff
@@ -193,6 +193,19 @@ public function check($value, string $rule, array $errors = []): bool
 	 */
 	protected function processRules(string $field, string $label = null, $value, $rules = null, array $data)
 	{
+		// If the if_exist rule is defined...
+		if (in_array('if_exist', $rules))
```

Comment:
I think we could check if the value is "empty" #830 around this lines.

So #614 should not be implemented, as this would mean that an empty string is a valid email, for example. But of course, I just suggest.